website/content/docs/upgrading/upgrade-to-1.10.x.mdx

```diff
@@ -77,6 +77,8 @@ naming collisions could result unexpected default behavior. Additionally, we rec
 the corresponding details in the OIDC provider [concepts](/docs/concepts/oidc-provider) document
 to understand how the built-in resources are used in the system.
 
+@include 'raft-panic-old-tls-key.mdx'
```

Comment:
Should this be under the Known Issues section for each of these pages?

Reply:
I'm trying to follow the existing pattern for embedding partials, even though I don't like it, because otherwise I'll have to change a bunch of things.